test/new/db/anal/path

```diff
@@ -12,7 +12,6 @@ EXPECT=<<EOF
 0x1000012c6
 0x1000012e7
 0x100001682
-0x10000168b
```

Comment:
Thats wrong.
1) The last bb should be also listed
2) an instruction in the middle of a bb should be a valid argument

Reply:
1 is trivial.
I explained on telegram why 2 is bad.